src/tabs/tabs.scss

```diff
@@ -4,100 +4,104 @@
 
 @mixin iui-tabs {
   @include iui-reset;
-
-  @include iui-font-family;
   display: flex;
+  align-items: center;
   list-style: none;
   padding: 0;
-  margin: round($iui-baseline/2) 0 0 0;
   user-select: none;
 
   > li {
     display: inline-block;
```

Comment:
These rules seem to do nothing now.

Reply:
You are correct, removed.

Reply:
What about `padding` and `margin`?

Reply:
Doesn't look like it is needed either.  I was also able to remove `padding: 0;` from `.iui-tabs` because `@include iui-reset;` handles that.